Android/app/src/main/java/app/intra/sys/firebase/RemoteConfig.java

```diff
@@ -87,4 +89,15 @@ public static boolean getUseSplitMode() {
       return true;
     }
   }
+
+  // Return any additional IPs known for this domain.
```

Comment:
Could explain the output format? Is it a list? You might want to parse it here, instead of moving a string list around as a string.

Reply:
I agree that it would normally be more elegant to parse it here, but since the source is a comma-separated list (because Firebase Remote Config doesn't support arrays), and the destination is a comma-separated list (because Gomobile doesn't support arrays of String), parsing and re-serializing it seems unnecessary.